### PR TITLE
Update Ibexa Fastly doc

### PR DESCRIPTION
**Pull request by andrerom**
Link to Ibexa Fastly doc seems broken as title has changed, also added reminder about applying Ibexa's VCL files, where to find them and why they are needed.

### CODE REVIEW

File: docs/src/frameworks/ibexa/fastly.md
Comment (andrerom) [inline docs/src/frameworks/ibexa/fastly.md:13]: obsolete release, so just dropped it

File: docs/src/frameworks/ibexa/fastly.md
Comment (andrerom) [inline docs/src/frameworks/ibexa/fastly.md:54]:
```suggestion
There are two VCL files provided as starting point for use Fastly with Ibexa DXP, you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`. They handle varying cache by user context hash _(permissions)_ as well as several other needs by Ibexa DXP and it's underlying HttpCache system.
```

File: docs/src/frameworks/ibexa/fastly.md
Comment (andrerom) [inline docs/src/frameworks/ibexa/fastly.md:13]:
```suggestion
In Ibexa DXP by default, Varnish is enabled by default when deploying on Platform.sh. In order to use Fastly, Varnish must be disabled:
```

**Comment by andrerom** [comment]
@Crell Who should review this?

File: docs/src/frameworks/ibexa/fastly.md
Comment (chadwcarlson) [inline docs/src/frameworks/ibexa/fastly.md:54]:
```suggestion
There are two VCL files provided as starting points for using Fastly with Ibexa DXP; you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`. They handle varying cache by user context hash _(permissions)_ as well as several other needs by Ibexa DXP and it's underlying HttpCache system.
```

**Comment by chadwcarlson** [comment]
LGTM @andrerom - one nit, but Ill merge once that passes.

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**
